packages/sui-domain/scripts/postpublish.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path')
+const fs = require('fs')
+const CWD = process.cwd()
+
+const EXTENSIONS_TO_REMOVE = ['.js']
+
+const handleError = err => { if (err) throw err }
+
+fs.readdir(CWD, (err, files) => {
+  handleError(err)
+
+  files.forEach(file => {
+    const ext = path.extname(file)
+    EXTENSIONS_TO_REMOVE.includes(ext) &&
+      fs.unlink(path.join(CWD, file), handleError)
```

Comment:
I dont feel confortable removing file and copy files using CWD. I mean, we are totally sure that we dont remove users´s things ?!

Only to avoid put an extra folder in the require path

Reply:
Same here.

Reply:
We have three options:

1. Just export everything as an object in order to do:
`import { Entity, ValueObject } from '@s-ui/domain'`

2. Fix scripts and use `__dirname` instead `cwd`, as it points to the file location and it's safer.

3. Just keep the creepy `lib` stuff. The reason why I don't support this is not only aesthetic is as a concepto. If you has to explain a developer to import just the ValueObject, you has to explain why they have to put the `lib` in the require path everytime. Also, it feels like it's an internal stuff of the library while it's not, it's the main core of the package. I would like to avoid this pattern as much as possible but if there's not another good enough solution I will assume it.

Reply:
I think that the priority is a clean public API. I really like the '@s-ui/domain/Entity'-like imports.
What about simply making more secure the prepublish and postpublish?
On postpublish we could do a safe delete: instead or traversing files in root dir you could read the files in lib and only delete in root what exist in lib and have the same content.
This way you get rid of @carlosvillu and have your flat package XD

Reply:
hahahahaha Sounds good. :D

Reply:
We will use: `import { Entity, ValueObject } from '@s-ui/domain'